e2e/src/api/specs/trash.e2e-spec.ts

```diff
@@ -0,0 +1,107 @@
+import { LoginResponseDto, getAllAssets } from '@immich/sdk';
+import { Socket } from 'socket.io-client';
+import { errorDto } from 'src/responses';
+import { apiUtils, app, asBearerAuth, dbUtils, wsUtils } from 'src/utils';
+import request from 'supertest';
+import { afterAll, beforeAll, describe, expect, it } from 'vitest';
+
+describe('/trash', () => {
+  let admin: LoginResponseDto;
+  let ws: Socket;
+
+  beforeAll(async () => {
+    apiUtils.setup();
+    await dbUtils.reset();
+    admin = await apiUtils.adminSetup({ onboarding: false });
+    ws = await wsUtils.connect(admin.accessToken);
+  });
+
+  afterAll(() => {
+    wsUtils.disconnect(ws);
+  });
+
+  describe('POST /trash/empty', () => {
+    it('should require authentication', async () => {
+      const { status, body } = await request(app).post('/trash/empty');
+
+      expect(status).toBe(401);
+      expect(body).toEqual(errorDto.unauthorized);
+    });
+
+    it('should empty the trash', async () => {
+      const { id: assetId } = await apiUtils.createAsset(admin.accessToken);
+      await apiUtils.deleteAssets(admin.accessToken, [assetId]);
+
+      const before = await getAllAssets(
+        {},
+        { headers: asBearerAuth(admin.accessToken) },
+      );
+
+      expect(before.length).toBeGreaterThanOrEqual(1);
```

Comment:
Should this be an exact equals?

Reply:
The tests don't reset the database between test cases so there are some assets already there